### ISSUE TITLE
NeuroData Active PR List

### ISSUE THREAD

**Issue by jshinm** [neurodata/neurodata.io#570]
The following is the active* list of current PR in progress (last updated 12/22/2021)

*Activity since 09/01/2021

<html xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/jongm/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/jongm/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">

</head>

<body link=blue vlink=purple>

Lead | Repository | PR Name | PR Url
-- | -- | -- | --
adam2392 | scikit-learn | [TEST PR] Adding oblique trees (i.e. Forest-RC) to cythonized tree module | https://github.com//neurodata/scikit-learn/pull/11
adam2392 | scikit-learn | [TEST PR] Oblique forests | https://github.com//neurodata/scikit-learn/pull/10
KhelmholtzR | ProgLearn | Add Streaming Function | https://github.com//neurodata/ProgLearn/pull/526
parthgvora | ProgLearn | UF enhanced parameters to match GRF | https://github.com//neurodata/ProgLearn/pull/453
adam2392 | scikit-learn | Tom/grid to graph 26 | https://github.com//neurodata/scikit-learn/pull/8
LizaNaydanova | ProgLearn | Added neural network scene segmentation tutorial. | https://github.com//neurodata/ProgLearn/pull/527
KevinWang905 | ProgLearn | unsupervised contrastive loss | https://github.com//neurodata/ProgLearn/pull/522
Dante-Basile | ProgLearn | Supervised Contrastive Loss | https://github.com//neurodata/ProgLearn/pull/518
waleeattia | ProgLearn | Exploring task unaware vs aware BTE and generalization error experiment | https://github.com//neurodata/ProgLearn/pull/517
kfenggg | ProgLearn | Add notebook &amp; functions for adversarial domain adaptation | https://github.com//neurodata/ProgLearn/pull/464
tliu68 | ProgLearn | Adding KNNClassificationDecider | https://github.com//neurodata/ProgLearn/pull/463
latasianguy | ProgLearn | Draft of SPORF | https://github.com//neurodata/ProgLearn/pull/454
jzheng17 | df-dn-paper | Implementation of Hyperparameter Tuning for the VIsion and Audition   Folders | https://github.com//neurodata/df-dn-paper/pull/64
jzheng17 | df-dn-paper | hyperparamter tuning | https://github.com//neurodata/df-dn-paper/pull/62
sampan501 | hyppo | update MGC to use scipy as reference implementation in tests | https://github.com//neurodata/hyppo/pull/231
sampan501 | hyppo | Added kernel goodness of fit | https://github.com//neurodata/hyppo/pull/206
nhahn7 | SDTF | ENH Prepare SynF for Gaussian Simulation experiments (updated code) | https://github.com//neurodata/SDTF/pull/42
tathey1 | brainlit | Update config.yml | https://github.com//neurodata/brainlit/pull/328
tathey1 | brainlit | Renaming lddmm registration files | https://github.com//neurodata/brainlit/pull/320
dallen02 | brainlit | time sleep | https://github.com//neurodata/brainlit/pull/295
tathey1 | brainlit | get angle method | https://github.com//neurodata/brainlit/pull/241
mlemyco | brainlit | multiple radius | https://github.com//neurodata/brainlit/pull/114
Laknath1996 | kdg | Label Noise Experiments for KDN | https://github.com//neurodata/kdg/pull/23
dependabot | talks | Bump node-sass from 3.3.3 to 4.14.1 in /big-data-science | https://github.com//neurodata/talks/pull/28
dlee0156 | bilateral-connectome | Adding test comparing triangle subgraph counts | https://github.com//neurodata/bilateral-connectome/pull/11
Lawreros | m2g | update func output tree | https://github.com//neurodata/m2g/pull/408
Lawreros | m2g | [WIP] Bunch of bug fixes and features | https://github.com//neurodata/m2g/pull/398
kellymarchisio | goat | Fixes #1 - fix outlier handling in qap/qapot | https://github.com//neurodata/goat/pull/2
JBaijot | neuroparc | Add files via upload | https://github.com//neurodata/neuroparc/pull/40
jovo | neuroparc | updated readme trivially | https://github.com//neurodata/neuroparc/pull/24
adam2392 | SPORF | [WIP] An implementation of discontiguous sampling of the SRerf variant;   we call it MTORF(?) | https://github.com//neurodata/SPORF/pull/353
adam2392 | SPORF | Bic fix urerf | https://github.com//neurodata/SPORF/pull/350
megh1241 | SPORF | Add Cython USPORF and outlier detection example | https://github.com//neurodata/SPORF/pull/349
MaiRajborirug | SPORF | update maintainer and update version to 2.0.5 | https://github.com//neurodata/SPORF/pull/347
MrAE | SPORF | Several suggestions on the distance metrics of SPORF that may boost the   performance on geodesic learning | https://github.com//neurodata/SPORF/pull/346
ShumingXu | SPORF | update python to dev v2.0.5.1 for clarity | https://github.com//neurodata/SPORF/pull/342
MrAE | SPORF | [WIP] Add two feature sampling methods for graphs | https://github.com//neurodata/SPORF/pull/337
j1c | SPORF | fix string -&gt; int | https://github.com//neurodata/SPORF/pull/336
dfrancisco1998 | notebooks | Simulations from papers for minka ZG comparison | https://github.com//neurodata/notebooks/pull/13
dfrancisco1998 | notebooks | Comparing GraphSAGE to CASC | https://github.com//neurodata/notebooks/pull/12
zeou1 | notebooks | ASE/LSE vs SVD/Spectral Comparison | https://github.com//neurodata/notebooks/pull/11
kareef928 | notebooks | Add experiments on spherical clusterings of DCSBM embeddings | https://github.com//neurodata/notebooks/pull/10
PerifanosPrometheus | notebooks | ZG vs minka comparison using OpenMl-cc2018 | https://github.com//neurodata/notebooks/pull/8
dfrancisco1998 | notebooks | clean up homepage, remove about | https://github.com//neurodata/notebooks/pull/4

</body>

</html>

**Comment by jovo** [comment]
@jshinm the priority functionalities are listed below, along with the POC (point of contact) for them:

stuff into sklearn
- UF (@rflperry) 
- binning (@PSSF23) 
- streaming (@PSSF23) 
- oblique (@adam2392)
- multivariate feature selection (@sampan501)
- AutoGMM (@tliu68) 


LDDMM into skimage (@vikramc1, @DevinCrowley)  

@asaadeldin11 did i miss anything?

**Comment by jovo** [comment]
@jshinm please make a schedule with each of the POC's to complete the PRs.  
of course, the timeline must be somewhat flexible due to its dependence on the core developers.
but our turn around time is not dependent on them, and that is the main factor that we are concerned about.

**Comment by jshinm** [comment]
@jovo Thank you for this comprehensive list. I will go ahead and get in touch with each of the POC and start making concrete plans and rough timelines for each project. Up to this point, we've made some progress on oblique tree where I ran a benchmark for @adam2392 and he'll use that to push forward the next round of PR into sklearn. I will follow up with Adam and the rest of POCs when I get back on the 5th
